chia/server/ws_connection.py

```diff
@@ -560,10 +569,12 @@ async def _send_message(self, message: Message) -> None:
             message, self.local_capabilities, self.peer_capabilities
         ):
             if not is_localhost(self.peer_host):
-                msg = f"Rate limiting ourselves. message type: {ProtocolMessageTypes(message.type).name}, "
-                f"peer: {self.peer_host}"
-                self.log.debug(msg)
-                self.log.addFilter(TimedDuplicateFilter(msg, 60))
```

Comment:
is the `TimedDuplicateFilter` used anywhere else, or can that be removed now?

Reply:
It is still used in the farmer.  The uses there are less concerning though I still hesitate with 1) the cleanup that 'should' be done and 2) the regexing of every debug message.  But those aren't 'real' issues over there like the use here with repeated registration was.